.github/workflows/ci.yml

```
@@ -231,7 +215,9 @@ jobs:
           echo "" >> $GITHUB_STEP_SUMMARY
           echo "| Check | Status |" >> $GITHUB_STEP_SUMMARY
           echo "|-------|--------|" >> $GITHUB_STEP_SUMMARY
-          echo "| Build & Test | ${{ needs.build.result == 'success' && '✅' || '❌' }} |" >> $GITHUB_STEP_SUMMARY
+          echo "| Build & Test | ${{ needs.build-and-test.result == 'success' && '✅' || '❌' }} |" >> $GITHUB_STEP_SUMMARY
           echo "" >> $GITHUB_STEP_SUMMARY
-          echo "### Matrix Results" >> $GITHUB_STEP_SUMMARY
-          echo "- Node 24: Complete" >> $GITHUB_STEP_SUMMARY
+          echo "### Platform Results" >> $GITHUB_STEP_SUMMARY
+          echo "- Linux: Complete" >> $GITHUB_STEP_SUMMARY
+          echo "- macOS: Complete" >> $GITHUB_STEP_SUMMARY
+          echo "- Windows: Complete (tests skipped)" >> $GITHUB_STEP_SUMMARY
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)